lib/seek/samples/sample_type_editing_constraints.rb

```diff
@@ -75,12 +75,30 @@ def allow_type_change?(attr)
         end
       end
 
+      def allow_isa_tag_change?(attr)
+        if attr.is_a?(SampleAttribute)
+            return false if inherrited?(attr)
+
+            attr = attr.accessor_name
+        end
+
+        if attr
+          all_blank?(attr)
+        else
+          true
+        end
+      end
+
       def refresh_cache
         do_analysis
       end
 
       private
 
+      def inherrited?(attr)
```

Comment:
misspelling, also could this not be a method and question to the sample attribute? `.inherited_from_template_attribute?`